tests/mesh_test.cc

```diff
@@ -826,7 +826,10 @@ TEST_CASE("Mesh is checked for 2D case", "[mesh][2D]") {
         SECTION("Check assign rotation matrices to nodes") {
           // Map of nodal id and euler angles
           std::map<mpm::Index, Eigen::Matrix<double, Dim, 1>> euler_angles;
-          // Euler angles
+          // Node 0 (non-existent) with Euler angles of 10 and 20  deg
```

Comment:
What do you mean by non-existent? if so why does it succeed?

Reply:
Typo, sorry